packages/core/src/helpers/import.ts

```diff
@@ -217,6 +217,10 @@ export function parseStImport(atRule: AtRule, context: string, diagnostics: Diag
         rule: atRule,
         context,
         keyframes: {},
```

Comment:
do we remove this in v5?

Reply:
yes, todo is in the `Imported` interface